src/mono/wasm/runtime/invoke-js.ts

```diff
@@ -336,8 +336,8 @@ export const importedModulesPromises: Map<string, Promise<any>> = new Map();
 export const importedModules: Map<string, Promise<any>> = new Map();
 
 export function dynamic_import(module_name: string, module_url: string): Promise<any> {
-    mono_assert(module_name, "Invalid module_name");
-    mono_assert(module_url, "Invalid module_name");
+    mono_assert(typeof module_name === "string", "module_name must be string");
```

Comment:
previously this would have also asserted that module_name's length was not 0, we're no longer checking that here. is it intended to allow "" as name or url?

Reply:
fixed, thanks